styles/index.css

```diff
@@ -0,0 +1,60 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins:200i,400&display=swap');
+
+
+* {
+    margin: 0px;
+    padding: 0px;
+    color: white;
+  }
+
+:root {
+  --color-white: #f3f3f3;
+  --color-darkblue: #1b1b32;
+  --color-darkblue-alpha: rgba(27, 27, 50, 0.8);
+  --color-green: #37af65;
+}
+  body {
+  background: linear-gradient(90deg, rgba(2,0,36,1) 0%, rgba(9,9,121,1) 35%, rgba(0,212,255,1) 100%);
+  font-family: 'Poppins', sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.4;
+  color: var(--color-white);
+  margin: 0;
+    z-index: -1;
+  background: var(--color-darkblue);
+  background-image: linear-gradient(
+      115deg,
+      rgba(58, 58, 158, 0.8),
+      rgba(136, 136, 206, 0.7)
+    ),
+    url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+  background-size: cover;
+  background-repeat: repeat;
+  background-position: center;
+   
+  }
+
+h1{
```

Comment:
For a better structuring the codes you will be using, You should not include the same tag selectors (h1) with different properties.

Reply:
- [ ] Please try to make the student understand the requested change, you can also highlight the whole property this kind of looks confusing, you can add please or kindly to your comments to make it more polite